### ISSUE TITLE
Add some simple tests for the slugbuilder

### ISSUE THREAD

**Issue by Cryptophobia** [teamhephy/slugbuilder#8]
Write some simple testing on the slugbuilder functionality. Maybe make a request to pull a very small app and see if the slugs are built successfully for the app using the slugbuilder image.

**Comment by kingdonb** [comment]
I opened https://github.com/teamhephy/workflow-e2e/issues/18 and found some existing tests that are at https://github.com/teamhephy/workflow-e2e/blob/master/tests/buildpacks_test.go

I'm going to try building and running them against the teamhephy repos and a minikube cluster

**Comment by Cryptophobia** [comment]
Yes, this e2e slugbuilder buildpack test is exactly what we need in order to verify changes in the slugbuilder/slugrunner/buildpacks do not break the example apps being deployed.

Let me know how it goes and maybe we can write up some scripts for quick testing for Travis CI.
